programs/editor/wodotexteditor.js

```diff
@@ -298,6 +301,7 @@ var Wodo = Wodo || (function () {
             //
             eventNotifier = new core.EventNotifier([
                 EVENT_UNKNOWNERROR,
+                EVENT_MODIFIEDCHANGED,
```

Comment:
Agreed also as non-native speaker that this is not perfect wording. Hm. Time for a step back:

So I would like to have a property in the sense of matching-persistent-made-state for the editor object, that refers to the currently loaded document. Coming from the libraries that I am used to I picked the term "modified" for that property, for which the setter/getter usually are `isModified` & `setModified(bool)`. Which is some irregular naming, usually the regular naming for setters and getters for a property `x` follows the pattern `setX` & `getX`/`x`. Given that irregularity, the event/signal on the change of that property seems to vary even more, I found `modificationChanged`, `modifiedFlagChanged`, `isModifiedChanged`, `modified`.
A variant to the term "modified" I found with OpenStep, there it is "edited" (or rather "documentEdited").  But still with irregular setter naming deflection, `isDocumentEdited`. Anyone knows any other approaches to naming?

As talked about in the initial comment, I meanwhile have changed my patch locally to use this set/get API: `setDocumentModified(boolean modified)` and `isDocumentModified(): boolean`. Still with the idea that the term "modified" in the API might be familiar to many.

So "UNMODIFIEDCHANGED" would no longer hold. For "PENDINGCHANGED" (and thus perhaps `setPending` & `pending`) I am not sure that would be easily understandable ("what is pending?").

For now, with the changed API in mind, I would propose to go with 'EVENT_DOCUMENTMODIFIEDCHANGED`, mainly for consistency in term usage. But still hoping for something better... please? :)